lib/generators/sufia/templates/config/sufia.rb

```diff
@@ -120,7 +120,14 @@
   # NOTE: if you have always sent analytics to GA for downloads and page views leave this commented out
   # config.analytic_start_date = DateTime.new(2014,9,10)
 
-  config.geonames_username = ''
+  # Location autocomplete uses geonames to search for named regions.
+  # Specify the user for connecting to geonames:
+  # config.geonames_username = ''
+
+  # Should the acceptance of the licence agreement be active (checkbox), or
```

Comment:
You don't need to add this on upgrade. It's already defaulted to the old behavior.